### ISSUE TITLE
Request to Enable Discussions in Repository

### ISSUE THREAD

**Issue by damywise** [alexmercerind/media_kit#50]
I was wondering if it might be possible for you to enable discussions within this GitHub repository.

This would allow people to engage in Q&A sessions, share information about new releases or unexpected documentation, and showcase projects they've been working on. By providing this separate space for discussion, we could help prevent the issue tracker from becoming cluttered and ensure that issues are addressed in a more streamlined manner. It would also give you the opportunity to focus on more pressing matters, without feeling guilty about not responding to every comment or question that arises.

Would you be willing to consider enabling discussions within the repository?

**Comment by damywise** [comment]
In addition, should any discussions arise in the issue tracker, one could easily transfer them to the discussion space or close them as needed. This would help keep the issue tracker clean and organized, while also ensuring that important discussions are taking place in the appropriate forum.

**Comment by alexmercerind** [comment]
Ideally, I want this to be just _repository_ which keeps my code.

[alexmercerind/dart_vlc also has 'Discussions' enabled](https://github.com/alexmercerind/dart_vlc/discussions), experience has been same (issues still got spammed). Last 3:
- [Trying to sound smart while not knowing shit.](https://github.com/alexmercerind/dart_vlc/discussions/350)
- [Thinking that package will just work on macOS if they install VLC.](https://github.com/alexmercerind/dart_vlc/discussions/358)
- [Advertisement (?)](https://github.com/alexmercerind/dart_vlc/discussions/351)

I've already released my work under open-source license: use it; improve it; fix it. And it's a good deal (free).

As I said in the another issue comment, I have no interest in answering 10s of people on daily basis or do their job. If someone wants consultancy from me or a new feature for their professional project, they can reach out on a contract basis.

In complete honesty, my time is valuable as everyone else's 😢.

[audio_service](https://github.com/ryanheise/audio_service/blob/3f159397ec33d16ef22cc1fdad8f439dd5e0db1c/CONTRIBUTING.md#how-we-use-github) also asks developers to ask questions on StackOverflow instead & and that's exactly what it's used for.
[I'm pretty sure if people ask questions like this on StackOverflow, they're going to get bullied has hell]

<hr>

**TL;DR**

I think we should instead have a community (e.g. Discord Server) where people can answer each other, share information about new releases or unexpected documentation, and showcase projects they've been working on ... (like you said).

I don't think I can answer every individual personally alongside development process of package:media_kit (already done in hobby time). Having a Discord will save my GitHub & e-mail from redundant notifications, while repository will be used solely for development. If I find some conversation interesting, I can then participate in it.

So, how does Discord sound? Flutter team also has one.

**Comment by alexmercerind** [comment]
https://discord.gg/h7qf2R9n57

**Comment by damywise** [comment]
Discord server sounds really good! I actually thought to suggest that but for reasons I don't.

**Comment by alexmercerind** [comment]
I've published current work on pub.dev & included Discord links everywhere.
